### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by nickluijtgaarden** [summarization/semantic_summ#3]
Dear developers,

I have read your scientific paper and the tech is really interesting. Do you have any plans on making documentation available on this Github repository on how to use it?

I think this could really help in getting more attention to your paper.

Kind regards,
Nick

**Comment by jiangweisuc** [comment]
Yes, issue a note to explain how to  use it in detail , it will be greate
